### PR TITLE
made js for 2 modal windows

### PR DESCRIPTION
**Pull request by Vanderwise**

### CODE REVIEW

File: css/regstration.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
+@font-face {
+    font-family: 'Gilroylight';
+    src: url(.//fonts/gilroylight.woff2) format('woff2'),
```
Comment (tetenyov) [inline css/regstration.css]: remembered that woff2 should go first - good!

File: css/regstration.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    line-height: 24px;  
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    outline: none;
+    border: none;
+    background: linear-gradient(to top, transparent 16px, #f0c52e 16px, #f0c52e 24px, transparent 24px);  
+}
+.submit-button:hover {    
+    background-color: #f0c52e;
+}
```
Comment (tetenyov) [inline css/regstration.css]: typo in file name `regstration`

File: index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+var writeus = document.querySelector(".writeus-button");
+var modalform = document.querySelector(".modal-write");
+var fio = modalform.querySelector("[name=user-name]");
+var formclose = document.querySelector(".write-close");
+var form = modalform.querySelector("form");
+var email = modalform.querySelector("[name=user-email]");
+
+
+map.addEventListener("click", function (evt) {
+    evt.preventDefault();
```
Comment (tetenyov) [inline index.js]: AFAIR, no need need in preventDefault here

File: index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    modalmap.classList.add("map-show");
+});
+close.addEventListener("click", function (evt) {
+    evt.preventDefault();
```
Comment (tetenyov) [inline index.js]: and here

File: index.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+var map = document.querySelector(".contacts-map");
+var modalmap = document.querySelector(".modal-map");
```
Comment (tetenyov) [inline index.js]: use camelCase for naming of vars consisted of two or more words

File: index.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+var close = document.querySelector(".map-close");
+            
+var writeus = document.querySelector(".writeus-button");
```
Comment (tetenyov) [inline index.js]: camelCase

File: index.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+var modalform = document.querySelector(".modal-write");
```
Comment (tetenyov) [inline index.js]: camel

File: index.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+var fio = modalform.querySelector("[name=user-name]");
+var formclose = document.querySelector(".write-close");
```
Comment (tetenyov) [inline index.js]: camel

File: index.js [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    modalmap.classList.add("map-show");
+});
+close.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    modalmap.classList.remove("map-show");
+});
+window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+        if (modalmap.classList.contains("map-show")) {
+            evt.preventDefault();
```
Comment (tetenyov) [inline index.js]: what for?) what's the default behavior of click on Escape?

File: index.js [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+            modalmap.classList.remove("map-show");
+        }
+    }
+});           
+            
+writeus.addEventListener("click", function(evt) {
+    evt.preventDefault();
```
Comment (tetenyov) [inline index.js]: here it is reasonable, bc you don't make default action

Comment (tetenyov) [reply index.js]: please rethink preventDefaults below)

File: index.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+});
+window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+        if (modalform.classList.contains("form-show")) {
+            evt.preventDefault();
+            modalform.classList.remove("form-show");
+        }
+    }
+}); 
+
```
Comment (tetenyov) [inline index.js]: empty line))

File: css/regstration.css (hunk 1 above)
Comment (Vanderwise) [reply css/regstration.css]: I do :)

File: css/regstration.css (hunk 2 above)
Comment (Vanderwise) [reply css/regstration.css]: Corrected

File: index.js (hunk 3 above)
Comment (Vanderwise) [reply index.js]: Done

File: index.js (hunk 5 above)
Comment (Vanderwise) [reply index.js]: Done

File: index.js (hunk 6 above)
Comment (Vanderwise) [reply index.js]: Done

File: index.js (hunk 7 above)
Comment (Vanderwise) [reply index.js]: Done

File: index.js (hunk 8 above)
Comment (Vanderwise) [reply index.js]: Done

File: index.js (hunk 9 above)
Comment (Vanderwise) [reply index.js]: image
I took this from the lecture. If we press Esc and modalMap is already closed then what will we close? I unredstand that in this case we can do without Contains. Removed if and done it as in writeUs modal window

File: index.js (hunk 10 above)
Comment (Vanderwise) [reply index.js]: do you mean that evt have to be the first line after function?

File: index.js (hunk 11 above)
Comment (Vanderwise) [reply index.js]: is it ok here? In CSS it is OK :)

File: index.js (hunk 9 above)
Comment (ratetenyov) [reply index.js]: hmm, maybe here it is really reasonable 👍 I trust Academy)

File: index.js (hunk 11 above)
Comment (ratetenyov) [reply index.js]: it is not ok in all files

File: index.js (hunk 10 above)
Comment (ratetenyov) [reply index.js]: I meant here usage of preventDefault is needed)

File: map.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+map.addEventListener("click", function () {
+    modalMap.classList.add("map-show");
+});
+close.addEventListener("click", function () {
+    modalMap.classList.remove("map-show");
+});
+window.addEventListener("keydown", function(evt) {
+    evt.preventDefault();
+    modalMap.classList.remove("map-show");
+});
```
Comment (ratetenyov) [inline map.js]: empty line

File: form.js [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+});
+window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+        if (modalForm.classList.contains("form-show")) {
+            evt.preventDefault();
+            modalForm.classList.remove("form-show");
+        }
+    }
+}); 
+
```
Comment (ratetenyov) [inline form.js:31]: empty line

File: form.js [hunk 14]
```diff
@@ -0,0 +1,32 @@
+var writeUsButton = document.querySelector(".writeus-button");
+var modalForm = document.querySelector(".modal-write");
+var fio = modalForm.querySelector("[name=user-name]");
+var formClose = document.querySelector(".write-close");
+var form = modalForm.querySelector("form");
+var email = modalForm.querySelector("[name=user-email]");
+
+         
+writeUsButton.addEventListener("click", function(evt) {
+    evt.preventDefault();
```
Comment (ratetenyov) [inline form.js:10]: couldn't find this element in the code. What is the default action on click on this elements?

File: form.js [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+    modalForm.classList.add("form-show");
+    fio.focus();
+});
+formClose.addEventListener("click", function(evt) {
+    evt.preventDefault();
```
Comment (ratetenyov) [inline form.js]: same question as above. I just want you to understand for this preventDefaults are needed

**Review by ratetenyov — APPROVED** [review]

File: form.js (hunk 14 above)
Comment (Vanderwise) [reply form.js:10]: a class="writeus-button" href="#">Напишите нам</a
Here its a link and i prevent link's action

File: form.js (hunk 15 above)
Comment (Vanderwise) [reply form.js]: yes, here nothing to preventDefault. Removed evt.

File: form.js (hunk 13 above)
Comment (Vanderwise) [reply form.js:31]: done

File: map.js (hunk 12 above)
Comment (Vanderwise) [reply map.js]: done

**Pull request merged**